documentation/src/main/resources/openapi/ditto-api-2.yml

```diff
@@ -10466,7 +10466,7 @@ components:
           entityId: "org.eclipse.ditto:some-thing-1"
           hasPermissions: [ "READ" ]
         lamp_toggler:
-          resource: "message:/features/lamp/inbox/toggle"
+          resource: "message:/features/lamp/inbox/message/toggle"
```

Comment:
"message:/features/lamp/inbox/message**s**/toggle"

the plural "s" is missing ;)